net/sched/sch_skbprio.c

```diff
@@ -228,8 +214,6 @@ static int skbprio_dump(struct Qdisc *sch, struct sk_buff *skb)
 	struct skbprio_sched_data *q = qdisc_priv(sch);
 	struct tc_skbprio_qopt opt;
 
-	opt.limit = q->max_limit;
```

Comment:
Bug: we cannot drop this line, otherwise `opt` is being sent to userland without any information. Replace `q->max_limit` with `sch->limit`.